context/app/static/js/components/workspaces/WorkspaceToastSuccessMessage.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import Stack from '@mui/material/Stack';
+import Typography from '@mui/material/Typography';
+import Button from '@mui/material/Button';
+
+function WorkspaceToastSuccessMessage(id: number) {
+  return (
+    <Stack sx={{ maxWidth: '22rem' }}>
+      <Typography>
+        Workspace successfully launched in a new tab. If the tab didn&apos;t open, please check your pop-up blocker
+        settings and relaunch your workspace.
+      </Typography>
+      <Stack direction="row" justifyContent="flex-end" marginTop={1}>
```

Comment:
Do we need a second stack? Could we just apply these props to the outer stack and remove this? Perhaps add `alignItems="center"` for alignment?

Reply:
I believe the second `Stack` is necessary as the text and button are aligned respectively to the left and right sides of the `Toast`. There may be a different approach to this, though

Reply:
Try `justifyContent="space-between"`

Reply:
As a prop on the outer stack.

Reply:
You can use `alignSelf="flex-end"` to align the button on the right side. You might need to wrap the button in a `Box` but try it without first.